import_email_events.py

```diff
@@ -0,0 +1,17 @@
+#
+# Script to import "email event" metrics from S3 into redshift.
+#
+
+import import_events
+SCHEMA = """
+    type VARCHAR(30) NOT NULL,
+    locale VARCHAR(40),
+    domain VARCHAR(40),
+    template VARCHAR(64),
+    flow_id VARCHAR(64)
```

Comment:
> `flow_id` should be promoted to `DISTKEY` here...

Well, maybe it should. It depends on what the queries against this table will look like, but I presume you are going to join on `flow_id`.